src/profile-logic/marker-data.js

```diff
@@ -473,9 +473,14 @@ export function mergeStartAndEndNetworkMarker(markers: Marker[]): Marker[] {
             marker.data.status === 'STATUS_START'
               ? [marker, markerNext]
               : [markerNext, marker];
+          const startData: NetworkPayload = (startMarker.data: any);
+          const endData: NetworkPayload = (endMarker.data: any);
           const mergedMarker = {
-            data: endMarker.data,
-            dur: endMarker.dur,
+            data: {
+              ...endData,
+              startTime: startData.startTime,
+            },
```

Comment:
This isn't the end of the road, because we'll want to add a property to represent the startMarker's `endTime`. I think handling this would fix #1349 too.

but at least we now display the right durations.